resources/views/product/create.blade.php

```diff
@@ -0,0 +1,125 @@
+              <input id="name" class="form-control col-md-7 col-xs-12" data-validate-length-range="2" data-validate-words="1" name="name" placeholder="@lang('products.label_name_product')" required="required" type="text" >
+            </div>
+          </div>
+          <div class="item form-group">
+            <label class="control-label col-md-3 col-sm-3 col-xs-12" for="categories">@lang('products.label_categories_product')
+              <span class="required">@lang('products.required_product')</span>
+            </label>
+            <div class="col-md-6 col-sm-6 col-xs-12">
+              <select autofocus name="category">
+                  @foreach($categories as $category)
+                    <option value="{{ $category->id }}">{{ $category->name }}</option>
+                  @endforeach
+              </select>
+            </div>
+          </div>
+          <div class="item form-group">
+            <label class="control-label col-md-3 col-sm-3 col-xs-12" for="number"> @lang('products.label_price_product') <span class="required">@lang('products.required_product')</span>
+            </label>
+            <div class="col-md-6 col-sm-6 col-xs-12">
+              <input type="number" id="number" name="price" required="required" data-validate-min="1,200000" class="form-control col-md-7 col-xs-12" placeholder="000.000 vnd">
+            </div>
+          </div>
+          <div class="item form-group">
+            <label class="control-label col-md-3 col-sm-3 col-xs-12" for="number">@lang('products.label_remaining_amount_product') <span class="required">@lang('products.required_product')</span>
+            </label>
+            <div class="col-md-6 col-sm-6 col-xs-12">
+              <input type="number" id="number" name="remaining_amount" required="required" data-validate-minmax="1,800" class="form-control col-md-7 col-xs-12" placeholder="@lang('products.label_remaining_amount_product')">
+            </div>
+          </div>
+          <div class="item form-group">
+            <label for="descrition" class="control-label col-md-3 col-sm-3 col-xs-12">@lang('products.label_description_product')
+            </label>
+            <div class="col-md-6 col-sm-6 col-xs-12">
+              <textarea id="description" type="text" name="description" data-validate-length-range="6,32" class="form-control col-md-7 col-xs-12" placeholder="Description"></textarea> 
+            </div>
+          </div>
+           <div class="item form-group">
+            <label class="control-label col-md-3 col-sm-3 col-xs-12" for="is_on_sale">@lang('products.label_is_on_sale_product')
+              <span class="required">@lang('products.required_product')</span>
+            </label>
+            <div class="col-md-6 col-sm-6 col-xs-12">
+              <select autofocus name="is_on_sale">
+                <option value="1">@lang('products.option_sale_Yes_product')</option>
+                <option value="0">@lang('products.option_sale_No_product')</option>
```

Comment:
Please avoid hardcode, the option values `0` and `1` should be defined as constants.